pyanaconda/ui/gui/__init__.py

```diff
@@ -783,7 +783,7 @@ def _instantiateAction(self, actionClass):
         return obj
 
     def run(self):
-        (success, args) = Gtk.init_check(None)
+        (success, _args) = Gtk.init_check(None)
```

Comment:
I'm afraid that could easily get mixed-up with the `_()` localization function. Also, even though the second part of the Gtk.init_check() output is currently not used it could still be useful to know what it contains.